input/examples/practitioner-chau-fryer.xml

```diff
@@ -1,9 +1,34 @@
 <?xml version="1.0" encoding="UTF-8"?>
+<!-- Compliant with HI Vendor Test Data -->
 <Practitioner xmlns="http://hl7.org/fhir">
     <id value="chau-fryer"/>
     <meta>
         <profile value="http://hl7.org.au/fhir/core/StructureDefinition/au-core-practitioner"/>
     </meta>
+    <identifier>
+        <type>
+            <coding>
+                <system value="http://terminology.hl7.org/CodeSystem/v2-0203"/>
+                <code value="NPI"/>
+                <display value="National provider identifier"/>
+            </coding>
+            <text value="HPI-I"/>
+        </type>
+        <system value="http://ns.electronichealth.net.au/id/hi/hpii/1.0"/>
+        <value value="8003616566719020"/>
+    </identifier>
+    <identifier>
```

Comment:
remove aphra. If you want to demonstrate some sort of other identifier, make a local one.

Reply:
Done